### DISCUSSION TITLE
mempool list

### DISCUSSION THREAD

**Discussion by gushome7** [Chia-Network/chia-blockchain#4571] [Support]
Hi! Is there a way to list the mempool from command line? Can´t find anything in the docs.

**Comment by kimsk** [comment]
try `cdv rpc mempool`

**Comment by BrandtH22** [comment]
@gushome7 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
